source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -268,20 +314,23 @@ export class Carousel extends Container {
             "swiper-button-prev",
             "ac-carousel-left"
         );
+        prevElementDiv.style.zIndex = "20";
```

Comment:
what is this for?

Reply:
The carousel peers cover the navigation arrows, so updating the zIndex places them in front.